doc/man/sconsign.xml

```diff
@@ -94,17 +94,21 @@ was specified by the
 function).
 Any
 <emphasis>file</emphasis>
-argument that does not end in
-<markup>.dbm</markup>
+argument that ends in
+<markup>.dblite</markup>
 is assumed to be a traditional
 <markup>.sconsign</markup>
 file containing the signature entries
 for a single directory.
+If neither of those is true,
+<command>sconsign</command>
+attempts to guess the format (but will
```

Comment:
Remove will probably get it wrong..

Reply:
No problem, but it's true.  sconsign doesn't seem to know how to recognize the default sconsign "dblite" format in any meaningful way, it *only* recognizes the presence of the file extension. If not found, the effort is passed on to python's whidhdb, which didn't seem to be able to guess.  The sentence was intended to lead into the (existing) next sentence, which is that you can specify the format with an argument.